### ISSUE TITLE
Rolling back Zoom licenses in favor of host keys

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#2955]
Now that we understand that host keys are the same as having the credentials, we can cut down on expecting our chairs/TLs/others that run meetings to be zoom admins. SIGs/WGs no longer need licenses, especially if they don't want one, as we can centralize this service. Right now, we are asking 70+ people to understand how to administratively set up a lot of security details when they should focus on running the meeting and that's it. The best practices doc is 10+ pages of admin details. 

Organizers will still need to turn off the screen sharing in meeting settings as the only roll over responsibility. Zoom is working on making this a global setting vs individual.

Phase 1  - Before EOY
All SIGs/WGs that are still using their personal meeting ID from #2536 will have a new meeting created under our global account. This will generate a new zoom link in which will need to be updated. 

Phase 2 - Q1 2019
All SIGs/WGs organizers who no longer want to be responsible for a license can fold into the new structure

Phase 3 - Q1 2019
Everyone else

NonGoals
Cutting off everyones license right now
Disrupting your current meetings - especially those who are running them correctly

Docs that will be updated:
Zoom Guidelines
SIG Governance (soon to be home of sig-lifecycle.md)

Related:
#2923 
#2536 
[brendan's email to k-dev about zoom problems from the summer]

This issue will close after Phase 1. I'd like to hear Chair/TL/Meeting Organizer opinions on this in the comments.

**Comment by parispittman** [comment]
/close

if the current zoom admins want to fully roll this out, they can pick it up at another time.
